### ISSUE TITLE
SummaryOuterClass$MechanismType ClassNotFoundException

### ISSUE THREAD

**Issue by replomancer** [OpenMined/clojure-dp#7]
## Description
Trying autocomplete in the repl raises this exception:
```
ClassNotFoundException com.google.differentialprivacy.SummaryOuterClass$MechanismType
```
which refers to a protobuf type.

## How to Reproduce
Start the repl with `lein repl`, type `(b` and press `Tab` for autocomplete :boom:  

## System Information
 - OS: Ubuntu
 - OS Version: 20.04
 - Language Version: Clojure 1.8

## Additional Context
Same issue arises when I try various ways of mocking Noise for testing. I am probably mocking incorrectly, but it's always the same message about SummaryOuterClass$MechanismType which is return type in one of the methods of Noise interface.

I believe this issue results from the way I naively put the bazel built jar with the Java library into Maven repositories. A Maven release of the Java lib would clarify this.

**Comment by replomancer** [comment]
Update:
Installing libsummary-proto-speed.jar in local Maven repo and updating dependencies solves both problems: repl autocomplete works and I can reify Noise for testing purposes. PR is coming soon.

**Comment by replomancer** [comment]
Resolved by #15
